code/modules/antagonists/changeling/datum_changeling.dm

```diff
@@ -149,7 +149,6 @@
  * If they have two objectives as well as absorb, they must survive rather than escape.
  */
 /datum/antagonist/changeling/give_objectives()
-	add_antag_objective(/datum/objective/absorb)
```

Comment:
Remove this objective altogether then please

Reply:
I'd have no problem removing it all together but typically 'out of rotation' objectives have been kept, ala protect, download, etc. Has that view changed?

Reply:
IMO since its an objective for a specific antagonist, yes. Most of the out of rotation could really apply to any antag, and I don't really ever see an admin adding this objective to someone.

Reply:
Done.